docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube_zh.md

```diff
@@ -227,94 +250,156 @@ podSecurityPolicy:
   enabled: false
 ingressClassName: nginx
 modules:
-  - rollsite
-  - clustermanager
-  - nodemanager
-  - mysql
   - python
+  - mysql
   - fateboard
   - client
+  - spark
+  - hdfs
+  - nginx
+  - pulsar
 
-backend: eggroll
+computing: Spark
+federation: Pulsar
+storage: HDFS
+algorithm: Basic
+device: CPU
 
 ingress:
   fateboard:
     hosts:
     - name: party9999.fateboard.example.com
-  client:  
+  client:
     hosts:
     - name: party9999.notebook.example.com
-
-rollsite:
-  type: NodePort
-  nodePort: 30091
-  partyList:
-  - partyId: 10000
-    partyIp: 192.168.100.123
-    partyPort: 30101
+  spark:
+    hosts:
+    - name: party9999.spark.example.com
+  pulsar:
+    hosts:
+    - name: party9999.pulsar.example.com
 
 python:
   type: NodePort
   httpNodePort: 30097
   grpcNodePort: 30092
   logLevel: INFO
 
-servingIp: 192.168.100.123
+servingIp: 10.182.137.144
 servingPort: 30095
-```
 
-对于`/kubefate/examples/party-10000/cluster.yaml`，我们可以将其修改如下：
+nginx:
+  type: NodePort
+  httpNodePort: 30093
+  grpcNodePort: 30098
+  route_table:
+    10000:
+      fateflow:
+        - host: 10.182.137.144
+          http_port: 30103
+          grpc_port: 30108
+
+pulsar:
+  type: NodePort
+  httpNodePort: 30094
+  httpsNodePort: 30099
+  publicLB:
+    enabled: false
+  route_table:
+    9999:
+      host: pulsar
+      port: 6650
+      sslPort: 6651
+    10000:
+      host: 10.182.137.144
+      port: 30104
+      sslPort: 30109
+      proxy: ""
+```
+
+对于fate-10000，我们可以将其修改如下：
 ```
 name: fate-10000
 namespace: fate-10000
 chartName: fate
 chartVersion: v1.9.0
 partyId: 10000
-registry: "hub.c.163.com/federatedai"
+registry: ""
 pullPolicy:
 imagePullSecrets:
 - name: myregistrykey
 persistence: false
 istio:
   enabled: false
+ingressClassName: nginx
 podSecurityPolicy:
   enabled: false
-ingressClassName: nginx
 modules:
-  - rollsite
-  - clustermanager
-  - nodemanager
-  - mysql
   - python
+  - mysql
   - fateboard
   - client
+  - spark
+  - hdfs
+  - nginx
+  - pulsar
 
-backend: eggroll
+computing: Spark
+federation: Pulsar
+storage: HDFS
+algorithm: Basic
+device: CPU
 
 ingress:
   fateboard:
     hosts:
     - name: party10000.fateboard.example.com
-  client:  
+  client:
     hosts:
     - name: party10000.notebook.example.com
-
-rollsite:
-  type: NodePort
-  nodePort: 30101
-  partyList:
-  - partyId: 9999
-    partyIp: 192.168.100.123
-    partyPort: 30091
+  spark:
+    hosts:
+    - name: party10000.spark.example.com
+  pulsar:
+    hosts:
+    - name: party10000.pulsar.example.com
 
 python:
   type: NodePort
   httpNodePort: 30107
   grpcNodePort: 30102
   logLevel: INFO
 
-servingIp: 192.168.100.123
+servingIp: 10.182.137.144
```

Comment:
Please 192.168.xxx.xxx, because there is a security check in Webank to verify if the fake IP in document.